```
/usr/bin/c++ -DLIBREMIDI_ALSA -DLIBREMIDI_EXPORTS -DLIBREMIDI_HAS_UDEV -DLIBREMIDI_NO_BOOST -I/mnt/vcpkg-ci/buildtrees/libremidi/x64-linux-dbg -I/mnt/vcpkg-ci/buildtrees/libremidi/src/v4.2.1-604e663e2a.clean -I/mnt/vcpkg-ci/buildtrees/libremidi/src/v4.2.1-604e663e2a.clean/include -fPIC -g -std=gnu++20 -fPIC -MD -MT CMakeFiles/libremidi.dir/include/libremidi/midi_in.cpp.o -MF CMakeFiles/libremidi.dir/include/libremidi/midi_in.cpp.o.d -o CMakeFiles/libremidi.dir/include/libremidi/midi_in.cpp.o -c /mnt/vcpkg-ci/buildtrees/libremidi/src/v4.2.1-604e663e2a.clean/include/libremidi/midi_in.cpp
In file included from /mnt/vcpkg-ci/buildtrees/libremidi/src/v4.2.1-604e663e2a.clean/include/libremidi/backends/alsa_raw/helpers.hpp:2,
                 from /mnt/vcpkg-ci/buildtrees/libremidi/src/v4.2.1-604e663e2a.clean/include/libremidi/backends/alsa_raw/midi_in.hpp:3,
                 from /mnt/vcpkg-ci/buildtrees/libremidi/src/v4.2.1-604e663e2a.clean/include/libremidi/backends/alsa_raw.hpp:2,
                 from /mnt/vcpkg-ci/buildtrees/libremidi/src/v4.2.1-604e663e2a.clean/include/libremidi/backends.hpp:15,
                 from /mnt/vcpkg-ci/buildtrees/libremidi/src/v4.2.1-604e663e2a.clean/include/libremidi/midi_in.cpp:5:
/mnt/vcpkg-ci/buildtrees/libremidi/src/v4.2.1-604e663e2a.clean/include/libremidi/backends/linux/alsa.hpp:5:10: fatal error: alsa/asoundlib.h: No such file or directory
    5 | #include <alsa/asoundlib.h>
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
```

hmmmm this is strange, since alsa indeed is supposed to install that stuff:

```
alsa:x64-linux:/debug/lib/libasound.a
alsa:x64-linux:/debug/lib/libatopology.a
alsa:x64-linux:/debug/lib/pkgconfig/alsa-topology.pc
alsa:x64-linux:/debug/lib/pkgconfig/alsa.pc
alsa:x64-linux:/include/alsa/asoundef.h
alsa:x64-linux:/include/alsa/asoundlib.h               <-- !!!
alsa:x64-linux:/include/alsa/conf.h
alsa:x64-linux:/include/alsa/control.h
alsa:x64-linux:/include/alsa/control_external.h
alsa:x64-linux:/include/alsa/control_plugin.h
alsa:x64-linux:/include/alsa/error.h
alsa:x64-linux:/include/alsa/global.h
alsa:x64-linux:/include/alsa/hwdep.h
alsa:x64-linux:/include/alsa/input.h
alsa:x64-linux:/include/alsa/mixer.h
alsa:x64-linux:/include/alsa/mixer_abst.h
alsa:x64-linux:/include/alsa/output.h
alsa:x64-linux:/include/alsa/pcm.h
alsa:x64-linux:/include/alsa/pcm_external.h
alsa:x64-linux:/include/alsa/pcm_extplug.h
alsa:x64-linux:/include/alsa/pcm_ioplug.h
alsa:x64-linux:/include/alsa/pcm_old.h
alsa:x64-linux:/include/alsa/pcm_plugin.h
alsa:x64-linux:/include/alsa/pcm_rate.h
alsa:x64-linux:/include/alsa/rawmidi.h
alsa:x64-linux:/include/alsa/seq.h
alsa:x64-linux:/include/alsa/seq_event.h
alsa:x64-linux:/include/alsa/seq_midi_event.h
alsa:x64-linux:/include/alsa/seqmid.h
alsa:x64-linux:/include/alsa/sound/asoc.h
alsa:x64-linux:/include/alsa/sound/asound_fm.h
alsa:x64-linux:/include/alsa/sound/emu10k1.h
alsa:x64-linux:/include/alsa/sound/hdsp.h
alsa:x64-linux:/include/alsa/sound/hdspm.h
alsa:x64-linux:/include/alsa/sound/sb16_csp.h
alsa:x64-linux:/include/alsa/sound/sscape_ioctl.h
alsa:x64-linux:/include/alsa/sound/tlv.h
alsa:x64-linux:/include/alsa/sound/type_compat.h
alsa:x64-linux:/include/alsa/sound/uapi/asoc.h
alsa:x64-linux:/include/alsa/sound/uapi/asound_fm.h
alsa:x64-linux:/include/alsa/sound/uapi/emu10k1.h
alsa:x64-linux:/include/alsa/sound/uapi/hdsp.h
alsa:x64-linux:/include/alsa/sound/uapi/hdspm.h
alsa:x64-linux:/include/alsa/sound/uapi/sb16_csp.h
alsa:x64-linux:/include/alsa/sound/uapi/sscape_ioctl.h
alsa:x64-linux:/include/alsa/sound/uapi/tlv.h
alsa:x64-linux:/include/alsa/timer.h
alsa:x64-linux:/include/alsa/topology.h
alsa:x64-linux:/include/alsa/ump.h
alsa:x64-linux:/include/alsa/ump_msg.h
alsa:x64-linux:/include/alsa/use-case.h
alsa:x64-linux:/include/alsa/version.h
alsa:x64-linux:/include/asoundlib.h
alsa:x64-linux:/include/sys/asoundlib.h
alsa:x64-linux:/lib/libasound.a
alsa:x64-linux:/lib/libatopology.a
alsa:x64-linux:/lib/pkgconfig/alsa-topology.pc
alsa:x64-linux:/lib/pkgconfig/alsa.pc
alsa:x64-linux:/share/alsa/aclocal/alsa.m4
alsa:x64-linux:/share/alsa/copyright
alsa:x64-linux:/share/alsa/usage
alsa:x64-linux:/share/alsa/vcpkg-cmake-wrapper.cmake
alsa:x64-linux:/share/alsa/vcpkg.spdx.json
alsa:x64-linux:/share/alsa/vcpkg_abi_info.txt
alsa:x64-linux:/tools/alsa/bin/aserver
alsa:x64-linux:/tools/alsa/debug/bin/aserver
```

Is our alsa `.pc` messed up or something, or is libremidi missing a `target_link_libraries` meant to add the include path?

--- a/ports/libremidi/portfile.cmake
+++ b/ports/libremidi/portfile.cmake
@@ -3,14 +3,21 @@ vcpkg_from_github(
     REPO jcelerier/libremidi
     REF "v${VERSION}"
 
-    SHA512 7b73d5e1a565e9f85ac28fba041c66a151537a9205b4f3605fa70e18d5e651fa0562329853c7e08bf8e620f9d83d08d167b746d30c39bffcd325fbd6338d0538 
+    SHA512 de7092c70af6fc0a23c8e6018fbd9f380632ac9dec8794171726fda9a6e7ba45479a8e8317919ba7a8a0267524bab8d5430782a54bc50a914658cf277e18145b
     HEAD_REF master
 )
+
+vcpkg_list(SET options)
+if(VCPKG_TARGET_IS_LINUX)
+    vcpkg_list(APPEND options -DLIBREMIDI_NO_ALSA=OFF)
+else()
+    vcpkg_list(APPEND options -DLIBREMIDI_NO_ALSA=ON)
+endif()
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
-        -DLIBREMIDI_NO_ALSA=ON
+        ${options}
         -DLIBREMIDI_NO_BOOST=ON
         -DLIBREMIDI_NO_JACK=ON
 )

--- a/ports/libremidi/vcpkg.json
+++ b/ports/libremidi/vcpkg.json
@@ -1,11 +1,15 @@
 {
   "name": "libremidi",
-  "version": "4.1.0",
+  "version": "4.2.3",
   "port-version": 1,
   "description": "A modern C++ MIDI real-time & file I/O library",
   "homepage": "https://github.com/jcelerier/libremidi",
   "license": "BSD-2-Clause",
   "dependencies": [
+    {
+      "name": "alsa",
+      "platform": "linux"
+    },
     {
       "name": "vcpkg-cmake",
       "host": true

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4597,7 +4597,7 @@
       "port-version": 1
     },
     "libremidi": {
-      "baseline": "4.1.0",
+      "baseline": "4.2.3",
       "port-version": 1
     },
     "libressl": {

--- a/versions/l-/libremidi.json
+++ b/versions/l-/libremidi.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "4427d9259242a0dbc53753710b42b6531276f392",
+      "version": "4.2.3",
+      "port-version": 1
+    },
+    {
       "git-tree": "0a6cac97560ba37eae699bdb65f6561a8415d362",
       "version": "4.1.0",
       "port-version": 1

